packages/cubejs-query-orchestrator/src/orchestrator/PreAggregations.ts

```diff
@@ -1998,11 +1998,14 @@ export class PreAggregations {
   }
 
   public static targetTableName(versionEntry): string {
+    const dbCatalog = getEnv('dbCatalog');
+
+    const dbCatalogStringPart = dbCatalog ? `${dbCatalog}.` : '';
```

Comment:
You need to use the `versionEntry`, meaning the `dbCatalog` value should be used at the initial moment and not here.